scripts/generate-react-exports.js

```diff
@@ -77,6 +80,7 @@ for (const module of customElementsModules) {
   const Type = componentName + 'Type';
 
   importStatements.push(`import type ${Type} from "./${importPath}";`);
+  exportStatements.push(`export declare type ${Type.replace('Type', '')} = ${Type}`);
```

Comment:
You can use `componentName` variable instead of `${Type.replace('Type', '')}`.

Reply:
Nice catch, updated.